version.properties

```diff
@@ -1,2 +1,8 @@
-mockito.testng.version=1.0
+#Currently building Mockito version
 version=2.7.22
+
+#Used for generating notable release notes
+notableVersions=2.7.0, 2.6.1, 2.5.0, 2.4.0
```

Comment:
Oh this is nice, cleaner changelog. We still might want to maintain our own for communication purposes